CHANGELOG.md

```diff
@@ -2,7 +2,7 @@
 
 ## [v0.6.1](https://github.com/docker/notary/releases/tag/v0.6.0) 04/10/2018
 + Fixed bug where CLI requested admin privileges for all metadata operations, including listing targets on a repo [#1315](https://github.com/theupdateframework/notary/pull/1315)
-+ Prevented notary signer from being dumpable or ptraceable in Linux, except in debug mode [#1327](https://github.com/theupdateframework/notary/pull/1327)
++ Prevented notary signer from being dumpable or able to ptrace in Linux, except in debug mode [#1327](https://github.com/theupdateframework/notary/pull/1327)
```

Comment:
Per @justincormack's feedback, it should be "Prevented notary signer from being dumpable or being ptraced ..."